### ISSUE TITLE
"Update submodules after pull" script is not enabled but submudules are still updated on pull

### ISSUE THREAD

**Issue by bughit** [gitextensions/gitextensions#3476]
Is there some other setting that controls this?

**Comment by RussKie** [comment]
Is this still an issue in the current version?

**Comment by bughit** [comment]
yes, unless the enabled state of the script is intended for something else

**Comment by RussKie** [comment]
I can't replicate the issue in the canary.
Can you please test the following version and report back https://ci.appveyor.com/api/buildjobs/loqlwimjxwi6xcs3/artifacts/GitExtensions-2.50.90.1427-Mono.zip
Thanks

**Comment by bughit** [comment]
same thing, the update submodules script is disabled

![image](https://user-images.githubusercontent.com/895588/35062835-5aef8dd2-fb93-11e7-9684-1d2fc1aa93bb.png)

but a `"C:\Program Files\Git\bin\git.exe" submodule update --init --recursive` dialog opens after pull

**Comment by gerhardol** [comment]
![image](https://user-images.githubusercontent.com/6248932/35073876-589940fc-fbeb-11e7-86d8-757f2b42a83e.png)

**Comment by bughit** [comment]
That's a pretty confusing setting. The language indicates it's for toggling confirmations, not the action itself.

To toggle the action and confirmation it either should be two checksboxes or three clearly labeled radios (no action, action with confirmation, action without confirmation), and the section should not be called Confirmations.

Then what is the "enabled" state of the script for?
